HomeW11/main.py

```diff
@@ -0,0 +1,32 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pylab
+
+plt.style.use('bmh')
+
+x = np.linspace(-11, 11, 100)
+
+pylab.subplot(2, 1, 1)
+
+a = x**3
+b = x * 50
+y = a - b
+pylab.plot(x, y);

```

Comment:
Татьяна, зачем же столько сложностей?
```suggestion
y1 = x ** 3 - x * 50
pylab.plot(x, y1)
```